Documentation/ceph-upgrade.md

```diff
@@ -337,7 +337,7 @@ The largest portion of the upgrade is triggered when the operator's image is upd
 When the operator is updated, it will proceed to update all of the Ceph daemons.
 
 ```sh
-kubectl -n $ROOK_OPERATOR_NAMESPACE set image deploy/rook-ceph-operator rook-ceph-operator=rook/ceph:v1.8.0
+kubectl apply -f operator.yaml
 ```
```

Comment:
I'm not sure this is quite the best approach. What if users have modified their own `operator.yaml`. I think it's better for us to identify the specific change they will need.

Reply:
Right, what if we keep the previous command for setting the v1.8 image, then add a **Note** below this that recommends to update the entire operator.yaml because of the updates to the volume mounts.